config/config.go

```diff
@@ -178,8 +177,9 @@ func (cfg *Metrics) validateAndLog() {
 type MetricsType string
 
 const (
-	MetricsNone   MetricsType = "none"
-	MetricsInflux MetricsType = "influx"
+	MetricsNone       MetricsType = "none"
+	MetricsInflux     MetricsType = "influx"
+	MetricsPrometheus MetricsType = "prometheus"
```

Comment:
There's no need to add a MetricsPrometheus IMHO. It's a legacy config we're moving away from.

Reply:
Makes sense. Removed.